File: Java/PetShop.java
public class PetShop {
    // Deklarasi variabel instance private
    private int id;         // Menyimpan ID produk
    private String name;    // Menyimpan nama produk
    private String category;// Menyimpan kategori produk
    private int price;      // Menyimpan harga produk
    
    // Constructor default
    public PetShop() {
        // Inisialisasi nilai default
        this.id = -1;
        this.name = "";
        this.category = "";
        this.price = 0;
    }
    
    // Constructor dengan parameter
    public PetShop(int id, String name, String category, int price) {
        // Inisialisasi nilai sesuai parameter
        this.id = id;
        this.name = name;
        this.category = category;
        this.price = price;
    }
    
    // Method untuk mengatur semua data sekaligus
    public void setData(int newId, String newName, String newCategory, int newPrice) {
        this.id = newId;
        this.name = newName;
        this.category = newCategory;
        this.price = newPrice;
    }
    
    // Method getter untuk mengambil nilai id
    public int getId() {
        return this.id;
    }
    
    // Method getter untuk mengambil nilai name
    public String getName() {
        return this.name;
    }
    
    // Method getter untuk mengambil nilai category
    public String getCategory() {
        return this.category;
    }
    
    // Method getter untuk mengambil nilai price
    public int getPrice() {
        return this.price;
    }
}

File: Python/PetShop.py
class PetShop:
    # Deklarasi variabel private
    __id = -1          # Menyimpan ID produk
    __name = ""        # Menyimpan nama produk
    __category = ""    # Menyimpan kategori produk
    __price = 0        # Menyimpan harga produk

    # Constructor dengan parameter default
    def __init__(self, id = -1, name = "", category = "", price = 0):
        # Menginisialisasi atribut dengan nilai parameter
        self.__id = id
        self.__name = name
        self.__category = category
        self.__price = price
    
    # Method untuk mengatur data produk
    def set_data(self, id, name, category, price):
        self.__id = id
        self.__name = name
        self.__category = category
        self.__price = price
    
    # Getter untuk mendapatkan ID produk
    def get_id(self):
        return self.__id
    
    # Getter untuk mendapatkan nama produk
    def get_name(self):
        return self.__name
    
    # Getter untuk mendapatkan kategori produk
    def get_category(self):
        return self.__category
    
    # Getter untuk mendapatkan harga produk
    def get_price(self):
        return self.__price